### PR TITLE
Update README.md with OH link

### PR DESCRIPTION
**Pull request by pamelafox**

### CODE REVIEW

**Review by multispark — CHANGES_REQUESTED** [review]
Can you capitalize "hours" and "time"?

File: README.md [hunk 1]
```diff
@@ -85,6 +85,14 @@ Read the [official rules](https://github.com/microsoft/RAG_Hack/blob/main/OFFICI
 | 9/12, 12:30 PM UTC    | [Create RAG applications with AI Toolkit VSCode Extension](https://hdxu.cn/zF74r) |
 | 9/14, 12:30 PM UTC    | [Intro to GraphRAG](https://hdxu.cn/moQUg) |
 
+## Office hours
```
Comment (pamelafox) [inline README.md:88]:
```suggestion
## Office Hours
```

File: README.md [hunk 2]
```diff
@@ -85,6 +85,14 @@ Read the [official rules](https://github.com/microsoft/RAG_Hack/blob/main/OFFICI
...
+
+For additional help with your hacks, you can drop by office hours in our AI Discord channel. Here are the office hours scheduled so far:
+
+
+| Day/time              | Topic/Hosts                              |
```
Comment (pamelafox) [inline README.md:93]:
```suggestion
| Day/Time              | Topic/Hosts                              |
```

File: README.md [hunk 3]
```diff
@@ -85,6 +85,14 @@ Read the [official rules](https://github.com/microsoft/RAG_Hack/blob/main/OFFICI
 | 9/12, 12:30 PM UTC    | [Create RAG applications with AI Toolkit VSCode Extension](https://hdxu.cn/zF74r) |
 | 9/14, 12:30 PM UTC    | [Intro to GraphRAG](https://hdxu.cn/moQUg) |
 
+## Office hours
+
+For additional help with your hacks, you can drop by office hours in our AI Discord channel. Here are the office hours scheduled so far:
```
Comment (pamelafox) [inline README.md:90]:
```suggestion
For additional help with your hacks, you can drop by Office Hours in our AI Discord channel. Here are the Office Hours scheduled so far:
```

**Pull request merged**